tests/cairo_files/test_instructions.cairo

```diff
@@ -0,0 +1,38 @@
+// SPDX-License-Identifier: MIT
+
+%lang starknet
+
+// Starkware dependencies
+from starkware.cairo.common.alloc import alloc
+from starkware.cairo.common.bool import TRUE, FALSE
+from starkware.cairo.common.cairo_builtins import HashBuiltin, BitwiseBuiltin
+
+// Local dependencies
+from utils.utils import Helpers
+from kakarot.model import model
+from kakarot.instructions import EVMInstructions
+from tests.utils.utils import TestHelpers
+
+@external
+func test__unknown_opcode{
+    syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_ptr, bitwise_ptr: BitwiseBuiltin*
+}() {
+    alloc_locals;
+    let (bytecode) = alloc();
+    let ctx: model.ExecutionContext* = TestHelpers.init_context(0, bytecode);
+    EVMInstructions.unknown_opcode(ctx);
```

Comment:
what does this test? that nothing "breaks"?

Reply:
The assertion is in the python file, we verify that an exception is raised. Having the test in the python instead of cairo is not ideal but I didn't find a way to catch exceptions in cairo tests